main.go

```diff
@@ -104,6 +103,17 @@ func run(l *log.Logger, out io.Writer, args arguments) error {
 		l.Println(string(output))
 	}
 
+	var out io.Writer
+	if args.outputFile == "-" {
+		out = os.Stdout
+	} else {
+		var err error
+		out, err = os.Create(args.outputFile)
```

Comment:
@wryun the file needs to be closed explicitly and also the error from `.Close()` should be checked to ensure the contents have been fully synced to disk. Otherwise exit with error.

Reply:
Otherwise the file may be partially written, program exit with 0 and... bad things will happen.

Reply:
https://github.com/wryun/rjsone/pull/11